### ISSUE TITLE
New app does not connect to the old system

### ISSUE THREAD

**Issue by reinarrow** [lidiapb/moonboard#9]
Apparently they have changed the protocol and now intermediate holds are named with the letter 'R' instead of 'P', according to [this issue](https://github.com/e-sr/moonboard/issues/62).

**Comment by reinarrow** [comment]
The new app introduces 'show move beta', @sintoris is right about just changing "P" to "R" during normal operation, but when show beta is on you get additional holds/colours:

"L" -> lefthand
"R" -> Righthand
"M" -> Match hands
"F" -> footholds

so you have S, R, L, M, F, E hold types now!

Easy enough to fix, I'm not sure what the right colours should be so I ended up just making them up!

**Comment by Robbe92** [comment]
hi,

I think this https://github.com/reinarrow/moonboard/blob/c57b38f490469e2486d81e78fbb14288368dd668/ble/moonboard_app_protocol.py#L20 is incorrect because in python oroperator is **or** not **||**

**Comment by reinarrow** [comment]
@Robbe92 that is completely right. I still did not release and test , that's why the issue is still open. Glad you noticed! Thanks a lot
